dvc/command/add.py

```diff
@@ -14,38 +15,40 @@ class CmdAdd(CmdBase):
     def run(self):
         try:
             if len(self.args.targets) > 1 and self.args.file:
-                raise DvcException("can't use '--file' with multiple targets")
-
-            for target in self.args.targets:
-                self.repo.add(
-                    target,
-                    recursive=self.args.recursive,
-                    no_commit=self.args.no_commit,
-                    fname=self.args.file,
+                raise RecursiveAddingWhileUsingFilename(
+                    "can't use '--file' with multiple targets"
```

Comment:
Isn't it the same as default message in `RecursiveAddingWhileUsingFilename`?

Reply:
essentially sort of yes. would you prefer

1. `can't use '--file' with multiple targets`
2. `using fname with recursive is not allowed.`
3. something else

Reply:
@casperdcl Would it hurt if we combine them into one message? Smth like
```
using fname with multiple targets or `-R|--recursive` is not allowed
```